### PR TITLE
Regenerate Video Intelligence v1beta2

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

**Comment by geigerj** [comment]
@quartzmo We've moved the location of the table in the code generator to the `doc/` directory. Could you verify if this is the correct fix for the `jsondoc` issues we were having earlier?

**Comment by quartzmo** [comment]
@geigerj I pulled the branch and built the jsondoc, and the content looks correct:

```sh
$ cat jsondoc/google/cloud/videointelligence/v1beta2.json 
{"id":"google/cloud/videointelligence/v1beta2","name":"V1beta2","title":["Google","Cloud","Videointelligence","V1beta2"],"description":"<h1 id=\"google-cloud-video-intelligence-api-contents\">Google Cloud Video Intelligence API Contents</h1>\n\n<table class=\"table\">\n  <thead>\n    <tr>\n      <th>Class</th>\n      <th>Description</th>\n    </tr>\n  </thead>\n...
```

File: google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb [hunk 1]
```diff
@@ -48,51 +48,55 @@ module Cloud
...
+        #   The OAuth scopes for this service. This parameter is ignored if
+        #   an updater_proc is supplied.
+        # @param client_config [Hash]
+        #   A Hash for call options for each method. See
+        #   Google::Gax#construct_settings for the structure of
+        #   this data. Falls back to the default config if not specified
+        #   or the specified config is missing data points.
+        # @param timeout [Numeric]
+        #   The default timeout, in seconds, for calls made through this client.
+        def self.new \
```
Comment (quartzmo) [inline google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb]: So not all of these options will be documented?

**Comment by geigerj** [comment]
Great, thanks for checking about the jsondoc!

File: google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb (hunk 1 above)
Comment (geigerj) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb]: Right. The undocumented options are deprecated and result in a warning if you use them.

File: google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb [hunk 2]
```diff
@@ -48,51 +48,55 @@ module Cloud
...
+            updater_proc: nil,
+            credentials: nil,
+            scopes: nil,
+            client_config: nil,
+            timeout: nil,
+            lib_name: nil,
+            lib_version: nil
+          kwargs = Hash[
+            *method(__method__).parameters
+              .select { |_, param| binding.local_variable_get(param) != nil }
```
Comment (blowmage) [inline google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb]: Unfortunately Binding#local_variable_get is not available in Ruby 2.0. This is why the build is failing for this PR.

Compare [Binding in 2.1](https://ruby-doc.org/core-2.1.0/Binding.html) to [Binding in 2.0](https://ruby-doc.org/core-2.0.0/Binding.html).

Comment (geigerj) [reply google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta2.rb]: Done.

**Comment by geigerj** [comment]
@blowmage Could you take another look? CI is passing now.

**Comment by blowmage** [comment]
Oh, sorry, was waiting for a green build and I missed the notification.

**Comment by blowmage** [comment]
@geigerj There is a discussion on GoogleCloudPlatform/google-cloud-ruby/#1181 discussing some issues with the links added in this PR. Should we manually fix these links so they work and we can release? Or should we wait for a new GAPIC generation? 

Also, do you want to regen the code for V1beta1 so it matches V1beta2?

**Comment by geigerj** [comment]
@blowmage You can wait for a new GAPIC generation before the release. I'm cherry-picking Eva's fix for the links in the code generator (currently under review).

**Comment by blowmage** [comment]
K, FWIW I think those links will be problematic because they always go to the latest release. If you are looking at a previous release (or master) and click those links they will jump you to latest. That might be frustrating for users who aren't expecting it to change the version they are looking at.

**Comment by geigerj** [comment]
@blowmage Noted. (I copied your note over to #1181.) I am OK with that for now because it seems preferable to showing users a blank page. But we should make a fix in the generator to use a relative link that won't jump to the latest version.

**Pull request merged**